src/components/rockets.js

```diff
@@ -1,8 +1,31 @@
-const Rockets = () => (
-  <>
-    <h1>My Rockets</h1>
-    <h3>Page Under Construction</h3>
-  </>
-);
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets } from '../redux/rockets/rocketslice';
+import RocketInformation from './rocketinfo';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const load = useSelector((state) => state.rocketstore.load);
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [load]);
```

Comment:
@osmanwako ,
Please, Include dispatch as a dependency in the useEffect dependency array or If you want the useEffect callback to run only once (when the component mounts) and not depend on any specific variable, you can remove the dependency array altogether. This way, the useEffect callback will run only once during the initial rendering of the component.